Week3/homework/6-step3.js

```diff
@@ -4,3 +4,16 @@ const arr2d = [[1, 2], [3, 4], [5, 6]];
 const arr3d = [[[1, 2], [3, 4]], [[5, 6], [7, 8]]];
 
 // add your solution here, or add a comment on how you would tackle this problem
+
+//Printing all the items of an array with 3 dimensions:
+
+let itemsOfArray = '';
+for (let i = 0; i < arr3d.length; i++) {
+    for (let j = 0; j < arr3d[i].length; j++) {
+        for (let k = 0; k < arr3d[i][j].length; k++) {
+            itemsOfArray += arr3d[i][j][k] + ' ';
+        }
+    }
+}
+
+console.log(itemsOfArray);
```

Comment:
Nice job.  This works as expected.  Can you think of a solution if you don't know the dimensions of the nesting?